Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -3588,9 +3611,10 @@ public struct TriggerState
             [FieldOffset(26)] private ushort m_InteractionIndex;
             [FieldOffset(28)] private float m_Magnitude;
             [FieldOffset(32)] private uint m_LastPerformedInUpdate;
-            [FieldOffset(36)] private uint m_LastCanceledInUpdate;
-            [FieldOffset(40)] private uint m_PressedInUpdate;
-            [FieldOffset(44)] private uint m_ReleasedInUpdate;
+            [FieldOffset(36)] private uint m_LastUnperformedInUpdate;
+            [FieldOffset(40)] private uint m_LastCanceledInUpdate;
+            [FieldOffset(44)] private uint m_PressedInUpdate;
+            [FieldOffset(48)] private uint m_ReleasedInUpdate;
```

Comment:
Isn't changing the field order of a struct with an explicit layout a breaking change?

```suggestion
            [FieldOffset(36)] private uint m_LastCanceledInUpdate;
            [FieldOffset(40)] private uint m_PressedInUpdate;
            [FieldOffset(44)] private uint m_ReleasedInUpdate;
            [FieldOffset(48)] private uint m_LastUnperformedInUpdate;
```

Reply:
As far as I could tell, no native code was using this struct and it's internal to the package, so it wasn't a breaking change. But I can go with this suggestion to minimize the diff.